op-e2e/actions/l2_batcher.go

```diff
@@ -133,7 +133,7 @@ func (s *L2Batcher) Buffer(t Testing) error {
 		if s.l2BatcherCfg.GarbageCfg != nil {
 			ch, err = NewGarbageChannelOut(s.l2BatcherCfg.GarbageCfg)
 		} else {
-			ch, err = derive.NewChannelOut()
+			ch, err = derive.NewChannelOut(s.l2BatcherCfg.MaxL1TxSize)
```

Comment:
```suggestion
			ch, err = derive.NewChannelOut(s.l2BatcherCfg.MaxL1TxSize-1)
```
That's because we prepend a version byte to the tx data.

Reply:
this already happens here: https://github.com/ethereum-optimism/optimism/blob/b72138c696f90af23b879338a3fb80e547e74ac1/op-batcher/batcher/driver.go#L90